client/src/components/assessments/instant/InstantAssessmentsContainerField.js

```diff
@@ -105,31 +105,58 @@ const InstantAssessmentsContainerField = ({
   formikProps,
   canRead,
   canWrite,
-  readonly
+  readonly,
+  showEntitiesWithoutAssessments
 }) => {
   const { values } = formikProps
+  function sortEntries(e1, e2) {
+    let e1HasAssessments = false
+    const e1entityInstantAssessments = e1.getInstantAssessments()
+    e1entityInstantAssessments.forEach(([ak, ac]) => {
+      const filteredAssessment = Model.filterAssessmentConfig(
+        ac,
+        e1,
+        relatedObject
+      )
+      if (
+        !_isEmpty(filteredAssessment.questions) ||
+        !_isEmpty(filteredAssessment.questionSets)
+      ) {
+        e1HasAssessments = true
+      }
+    })
+    if (e1HasAssessments) {
+      return 1
+    }
+    return -1
+  }
+  function getEntitiesWithAssessments(entity) {
+    const entityInstantAssessments = entity.getInstantAssessments()
+    let hasAssessments = false
+    entityInstantAssessments.forEach(([ak, ac]) => {
```

Comment:
I would use `return e1entityInstantAssessments.some(…)` instead of looping over all of them and setting a boolean variable.